### ISSUE TITLE
Formatting inside thread forms

### ISSUE THREAD

**Issue by PEZ** [weavejester/cljfmt#223]
Hi, I want something like:

```clojure
(cond-> foo
  :foo (as-> $
             bar))
```

to be formatted as if the threaded param was in place:

```clojure
(cond-> foo
   :foo (as-> $
          bar))
```

In #45 we learn that `cljfmt` does not consider the outer forms when formatting a form. But there is also a suggestion to use a rule for `->` like forms to deal with it. What would such a rule look like?

**Comment by weavejester** [comment]
I don't believe there's currently a way of doing this, unless you assume that `as->` will always be part of an outer thread, or you just have a blanket "always indent by 2" rule.
